lib/rubocop/comment_config.rb

```diff
@@ -11,7 +11,7 @@ class CommentConfig
     COPS_PATTERN = "(all|#{COP_NAMES_PATTERN})".freeze
 
     COMMENT_DIRECTIVE_REGEXP = Regexp.new(
-      ('\A# rubocop : ((?:dis|en)able)\b ' + COPS_PATTERN).gsub(' ', '\s*')
+      ('# rubocop : ((?:dis|en)able)\b ' + COPS_PATTERN).gsub(' ', '\s*')
```

Comment:
I just noticed the `gsub` and it makes sense now.

I don't like the `gsub` part. I don't think it contributes to the clarity. Changing that would require changing `COP_NAMES_PATTERN`, so I suppose that's out of scope.